core/templates/pages/contributor-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -179,7 +179,7 @@ export class ContributionOpportunitiesBackendApiService {
     });
   }
 
-  async fetchFeaturedTranslationLanguages():
```

Comment:
@KhushiGangopadhyay, I think there are other few functions in this file which needed to async, right? Can you please check?

Reply:
Ditto.

Reply:
done